packages/core/admin/admin/src/components/GuidedTour/Homepage/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useGuidedTour } from '@strapi/helper-plugin';
+import { useIntl } from 'react-intl';
+import { Stack } from '@strapi/design-system/Stack';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { Typography } from '@strapi/design-system/Typography';
+import { LinkButton } from '@strapi/design-system/LinkButton';
+import { Button } from '@strapi/design-system/Button';
+import ArrowRight from '@strapi/icons/ArrowRight';
+import StepperHomepage from './components/Stepper';
+import layout from '../layout';
+
+const GuidedTourHomepage = () => {
```

Comment:
Can you add some tests?

Reply:
we already added logic test for this component in Homepage tests but we could at least add a snapshot for the component!